documentation/oracle/governance/adding_a_price_identifier.md

```diff
@@ -0,0 +1,48 @@
+# Adding a Price Identifier
+
+## What is a price identifier?
+
+A [price identifier](../../synthetic_tokens/glossary.md#price-identifier) is a natural language descriptor of the reference index determining how much collateral is needed for a token sponsor to be properly collateralized. 
+Because DVM voters need to be able to vote on the value of this price identifier when disputes are raised, the DVM keeps a list of approved price identifiers. 
+
+An example of a price identifier is “GOLD_APR20”. 
```

Comment:
Describe this! Remember that an outsider reading `GOLD_APR20` will prob just scratch their head: what does the price identifier here actually do? What are voters expected to vote on?

Also I would pick something spot, not with an expiry date. So go with `GOLD_USD` or something like that.

Also, I think it'll look best if the identifier is formatted in code style like `this` and not quotes like "this".

Reply:
All of that information should be documented in an associated UMIP. Since we are going to provide a UMIP for ETHBTC, I think we should use that as our example.

Reply:
I'm a bit confused as to what you're looking for. Can you give me an example? I can then translate it to reflect the ETHBTC UMIP if helpful.

Reply:
I see your example below, now, and have committed it.